doc/man3/SSL_CTX_set_options.pod

```diff
@@ -122,12 +122,14 @@ being set as well. Always ignored on the client.
 Allow legacy insecure renegotiation between OpenSSL and unpatched clients or
 servers. See the B<SECURE RENEGOTIATION> section for more details.
 
-=item SSL_OP_CIPHER_SERVER_PREFERENCE
+=item SSL_OP_SERVER_PREFERENCE
 
-When choosing a cipher, use the server's preferences instead of the client
-preferences. When not set, the SSL server will always follow the clients
-preferences. When set, the SSL/TLS server will choose following its
-own preferences.
+When choosing a cipher, signature, (TLS 1.2) curve or (TLS 1.3) group, use the
+server's preferences instead of the client preferences. When not set, the SSL
+server will always follow the clients preferences. When set, the SSL/TLS server
+will choose following its own preferences.
+For backwards-compatibility, the old name SSL_OP_CIPHER_SERVER_PREFERENCE
+can still be used but is deprecated.
```

Comment:
I'm not sure I would use the word deprecated here (perhaps "discouraged" or something like that). The word deprecated implies a bit more (e.g. that you can't use it in a "no-deprecated" build)

Reply:
Perhaps there should also be some notes about this in the HISTORY section of this man page. I would also add an entry in CHANGES.md